cluster/cluster.go

```diff
@@ -119,8 +120,8 @@ func ListPods(client kubernetes.Interface) (*v1.PodList, error) {
 		metav1.ListOptions{FieldSelector: "status.phase=Running,metadata.namespace!=kube-system,metadata.namespace!=gke-gmp-system"},
 	)
 	if err != nil {
-		err = fmt.Errorf("error getting pods: %v", err)
-		return nil, err
+		// Log the error, but continue execution
+		log.Printf("Error listing pods: %v", err)
```

Comment:
If you just log the error then it will return `nil, nil` and will cause issues downstream. You should handle errors correctly where you are calling this function, not here